packages/pytest-simcore/src/pytest_simcore/services_api_mocks_for_aiohttp_clients.py

```diff
@@ -31,6 +36,7 @@
     "pytest_simcore.aioresponses_mocker",
 ]
 
+_FAKER = Faker()
```

Comment:
just saw that now
but there is a Faker fixture as well

Reply:
Do you know where it is located?

Reply:
you can just do
```python
from faker import Faker

def test_mytest(faker: Faker):
...
```

Reply:
Aha, perfect. Thanks a lot.